File: websocket/src/main/java/ee/taltech/jajann/websocket/dto/Position.java
package ee.taltech.jajann.websocket.dto;

import lombok.Data;

@Data
public class Position {
    private String assetId;
    private String symbol;
    private String exchange;
    private String assetClass;
    private Double avgEntryPrice;
    private Integer qty;
    private String side;
    private Double marketValue;
    private Double costBasis;
    private Double unrealizedPl;
    private Double unrealizedPlpc;
    private Double unrealizedIntradayPl;
    private Double unrealizedIntradayPlpc;
    private Double currentPrice;
    private Double lastdayPrice;
    private Double changeToday;

}

File: vue-electron-app/src/interfaces/Position.ts
export interface Position {
  symbol?: string;
  exchange?: string;
  asset_class?: string;
  avg_entry_price?: number;
  qty?: number;
  market_value?: number;
  cost_basis?: number;
  unrealized_pl?: number;
  unrealized_plpc?: number;
  unrealized_intraday_pl?: number;
  unrealized_intraday_plpc?: number;
  current_price?: number;
  lastday_price?: number;
  change_today?: number;
  account?: string;
}
